docs/backend-api/resources/billing/transaction.md

```diff
@@ -1,47 +1,97 @@
 ---
 title: Transaction
-description: Transaction
+description: API call to get list of user's billing transactions for the specified period.
 ---
 
 # Transaction
 
+API path: `/transaction`.
+
+API call to get user's billing transactions.
+
+## Transaction object
+
+```json
+{
+"description": "Recharge bonus balance during tracker registration",
+"type": "bonus_charge",
+"subtype": "register",
+"timestamp": "2021-01-28 08:16:40",
+"user_id": 12203,
+"dealer_id": 5001,
+"tracker_id": 303126,
+"amount": -10.0000,
+"new_balance": 800.0000,
+"old_balance": 810.0000,
+"bonus_amount": 10.0000,
+"new_bonus": 10.0000,
+"old_bonus": 0.0000
+}
+```
+
+* `description` - string. Transaction description.
+* `type` - string enum. Type of transaction.
+* `subtype` - string enum. Subtype of transaction.
+* `timestamp` - string date/time. When transaction created.
+* `user_id` - int. ID of a user which made a transaction.
+* `dealer_id` - int. ID of a dealer.
+* `tracker_id` - int. Tracker id. 0 if transaction not associated with tracker.
+* `amount` - double. Amount of money in transaction, can be negative. e.g. -10.0000 means 10 money units removed from user`s balance.
+* `new_balance` - double. User`s money balance after transaction.
+* `old_balance` - double. User`s money balance before transaction.
+* `bonus_amount` - double. Amount of bonus used in transaction, can be negative. e.g. 10.0000 means 10 bonuses units added to user`s bonus balance.
+* `new_bonus` - double. User`s bonus balance after transaction.
+* `old_bonus` - double. User`s bonus balance before transaction.
+
 ### list
 
-Get list of user's billing transactions for the specified period.
+Gets list of user's billing transactions for the specified period.
 
-**required subuser rights**: payment_create
+**required sub-user rights**: `payment_create`.
 
 #### parameters
 
-* **from** – date/time. Start date/time for searching.
-* **to** – date/time. End date/time for searching. must be after "from" date.
-* **limit** – int (optional). Maximum number of returned transactions.
+| name | description | type|
+| :------ | :------ | :-----|
+| from | Start date/time for searching. | string date/time |
+| to | End date/time for searching. Must be after `from` date. | string date/time |
```

Comment:
"string date/time" -> just "date/time". This type described in [Getting started](https://developers.navixy.com/backend-api/getting-started/#data-types).